sdk/dynamicconfig/internal/metricpattern/metricpattern.go

```diff
@@ -20,6 +22,8 @@ import (
 	pb "go.opentelemetry.io/contrib/sdk/dynamicconfig/internal/proto/experimental/metrics/configservice"
 )
 
+// Matches determines whether a name falls in the set of names prescribed by
```

Comment:
"in the set of metric names" add "metric"

Reply:
changed